docs/modules/databases/mongodb.md

```diff
@@ -0,0 +1,57 @@
+# Mongo DB Module
+
+!!! note
+    This module is INCUBATING. While it is ready for use and operational in the current version of Testcontainers, it is possible that it may receive breaking changes in the future. See [our contributing guidelines](/contributing/#incubating-modules) for more information on our incubating modules policy.
+
+# Java8 MongoDbContainer for constructing a single node MongoDB replica set. To construct a multi-node MongoDB cluster, consider the [mongodb-replica-set project](https://github.com/silaev/mongodb-replica-set/)              
+
+## Usage example
+
+The following example shows how to create a MongoDbContainer
+
+<!--codeinclude-->
+[Creating a MongoDB container](../../../modules/mongodb/src/test/java/org/testcontainers/containers/MongoDbContainerTest.java) inside_block:creatingMongoDbContainer
+<!--/codeinclude-->
+
+<!--codeinclude-->
+[Starting a MongoDB container](../../../modules/mongodb/src/test/java/org/testcontainers/containers/MongoDbContainerTest.java) inside_block:startingMongoDbContainer
```

Comment:
When rendered, it becomes a tab. Should be a separate code block (see other docs pages)

Reply:
done

Reply:
LGTM